vumi_message_store/batch_info_cache.py

```diff
@@ -179,12 +179,13 @@ def add_inbound_message_key(self, batch_id, message_key, timestamp):
             yield self.redis.incr(self.inbound_count_key(batch_id))
             yield self.truncate_inbound_message_keys(batch_id)
 
-    def add_from_addr(self, batch_id, from_addr):
+    def add_from_addr(self, batch_id, *from_addrs):
         """
         Add a from address to the HyperLogLog counter for the batch.
         """
         return self.redis.pfadd(
-            self.from_addr_key(batch_id), from_addr.encode('utf-8'))
+            self.from_addr_key(batch_id),
+            *[from_addr.encode('utf-8') for from_addr in from_addrs])
```

Comment:
This might be a little more readable if the encoding happened before the call:
```python
from_addrs = [from_addr.encode('utf-8') for from_addr in from_addrs]
return self.redis.pfadd(self.from_addr_key(batch_id), *from_addrs)
```